acceptance/launcher_test.go

```diff
@@ -27,36 +27,38 @@ func TestLauncher(t *testing.T) {
 }
 
 func testLauncher(t *testing.T, when spec.G, it spec.S) {
-	when("no args and no default", func() {
-		it("launches web", func() {
-			cmd := exec.Command("docker", "run", "--rm", launchImage)
-			output, err := cmd.CombinedOutput()
-			if err != nil {
-				t.Fatalf("failed to run %v\n OUTPUT: %s\n ERROR: %s\n", cmd.Args, output, err)
-			}
-			expected := "Executing web process-type"
-			if !strings.Contains(string(output), expected) {
-				t.Fatalf("failed to execute web:\n\t got: %s\n\t want: %s", output, expected)
-			}
+	when("there is no CMD provided", func() {
```

Comment:
I'm wondering if we can make this even clearer. If I'm understanding correctly, `CMD` is the argument(s) provided to the launcher. For a minute I was wondering if it was the `Cmd` from the OCI container config...

Reply:
I did mean it in the OCI sense. When launcher is the `entrypoint`, the arguments provided to launcher and the container `cmd` are one and the same. But I am happy to reword it from the perspective of `launcher` rather than the user who runs the container, if it's clearer.

Reply:
Ahhh you are exactly right... I think I confused myself.